.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  run_tests:
+    name: tests
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ['3.8']
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install pip
+        run: pip install -r requirements/pip.txt
+
+      - name: Install Dependencies
+        run: pip install -r requirements/travis.txt
+
+      - name: Run Tests
+        run: tox -e py
+
+  run_quality:
+    name: quality checks
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ['3.8']
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install pip
+        run: pip install -r requirements/pip.txt
+
+      - name: Install Dependencies
+        run: pip install -r requirements/travis.txt
+
+      - name: Run Quality Check
+        run: tox -e quality
```

Comment:
A thought on reducing some of the duplication here: could we have just one job (maybe "run_tox") with a matrix entry like `toxenv: [py, quality]`?  Then we could use an `env` section in the job with `TOXENV: ${{ matrix.toxenv }}`.  Alternatively, we could use [tox-gh-actions](https://pypi.org/project/tox-gh-actions/) to do something very similar; it would add a dependency, but seems like it would also handle tox configs for testing multiple Django/celery/DRF versions more easily.

Reply:
Ah, I forgot that one of the drawbacks of `tox-travis` and `tox-gh-actions` is that they only utilize one worker per Python version, rather than one per tox env.  Probably more efficient to stick with the `TOXENV` approach.

Reply:
I tried using (defaults)[https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#defaults] for GitHub workflows but it has its limitations